mpi-mediator/package-metadata.json

```diff
@@ -11,9 +11,23 @@
   ],
   "environmentVariables": {
     "STATEFUL_NODES": "single",
-    "CLIENT_REGISTRY_URL": "http://santedb-mpi:8080",
     "OPENHIM_MEDIATOR_URL": "https://openhim-core:8080",
-    "TRUST_SELF_SIGNED": true,
-    "REGISTER_MEDIATOR": true
+    "TRUST_SELF_SIGNED": "true",
+    "OPENHIM_USERNAME": "root@openhim.org",
+    "OPENHIM_PASSWORD": "instant101",
+    "REGISTER_MEDIATOR": "false",
```

Comment:
Should the default not be set to 'true'?

Reply:
I made it false because the openhim is not a dependency of the mpi-mediator